prov/verbs/src/fi_verbs.h

```diff
@@ -394,33 +443,177 @@ void fi_ibv_monitor_unsubscribe(struct ofi_mem_monitor *notifier,
 				struct ofi_subscription *subscription);
 struct ofi_subscription *fi_ibv_monitor_get_event(struct ofi_mem_monitor *notifier);
 
+/*
+ * An XRC SRQ cannot be created until the associated RX CQ is known,
+ * maintain a list of validated pre-posted receives to post once
+ * the SRQ is created.
+ */
+struct fi_ibv_xrc_srx_prepost {
+	struct slist_entry	prepost_entry;
+	void			*buf;
+	void			*desc;
+	void			*context;
+	size_t			len;
+	fi_addr_t		src_addr;
+};
```

Comment:
This doesn't handle an iovec.  I'm guessing that's okay, but it should be reported as a limitation through the rx_attr.

Reply:
Right. I have updated the RX attributes to reflect IOV limit when XRC protocol is used.